data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1615 @@
+			`	Eventually you get a message from Dusk indicating that they are all packed up and ready to begin loading.`
+			label wrapup
+			`	You find Dusk in the main hall that opens directly off of the bay where the <ship> is docked. Remnant scurry around stacking equipment in preparation for loading, and a team appears to be setting up a number of instruments around the periphery of the room. Dusk spots you, and waves you over to where he appears to be checking the readings on a console.`
+			choice
+				`	"What are you doing now?"`
+				`	"Ready to load up?"`
+					goto loading
+			`	Dusk smiles at you and gestures at the others setting up instruments around the edge of the room. "We are leaving these instruments here running automated data collection routines. We cannot run the main instruments without access to larger power sources such as your ship, but these smaller instruments will be able to record baseline data for us as well as documenting any sudden changes in some of the spectral emanations. They will provide useful data for us in the future." He smiles, then continues. "As the saying goes, the best time to start collecting data is a hundred years ago. The second best time to start collecting data is now."`
+			label loading
+			`	He gestures to the rest of the team, who immediately start loading the <ship>. "Well, it is time to head back." Dusk heads over to the equipment pile, and starts carrying equipment onboard.`
+	on complete
+		require "Research Laboratory"
+		payment 350000
+		clear "Remnant: Dusk Busy"
+		conversation
+			`The trip back to <destination> goes quickly but quietly, as all the Remnant seem to be focused on writing reports of their observations. From what you can tell, each member of the team writes up their own observations and experiences, and then Dusk assembles all the reports into an expedition report that includes each researcher's verbatim report as well as his own.`
+			`	The <ship> settles into the current landing pattern for the main spaceport on <destination> as you flip through the exterior camera feeds. In the distance, you notice that one of the research hangars has some activity going on around it, but otherwise, the spaceport seems to be humming along as usual. Dusk stops by the cockpit as you settle the <ship> into its berth. "Thank you for helping us, <first>. It will take us some time to analyze this data, but we will likely be mounting another expedition to Ssil Vida in the near future." He pauses, then adds, "I do not think we ever mentioned it, but, as is appropriate for your assistance, a logistics adjustment of <payment> has been applied to your account to cover your costs and provide resources to prepare for future endeavors." Dusk gives you a polite nod before heading off to help the other researchers unload the equipment.`
+
+
+
+mission "Remnant: Cognizance 22"
+	name "Deliver Data Backups for Taely"
+	description "Deliver a data backup to <stopovers> for Taely."
+	source
```

Comment:
This can be triggered on any of the three major worlds, and the return mission prior to this could end up at any of the three worlds too.

The `neighbor distance 1 5` is supposed to ensure the stopover is somewhere *other* than the planet it starts on, if that's what you were getting at.

Reply:
Huh. Not sure what happened last night, but worked fine for me today. Maybe my brain glitched out for a sec. Feel free to resolve once you've seen this!